migrations/20170206142200_setup.js

```diff
@@ -35,7 +35,7 @@ exports.up = function(knex, Promise) {
 		.createTable('keyword', table => {
 			table.increments('id');
 			table.integer('keyword_type_id').references('keyword_type.id').notNullable();
-			table.string('name').unique().notNullable();
```

Comment:
We still need the unique constraint here. I've opened a bug explaining why we're getting the name conflict. See #106